app/main.ipynb

```diff
@@ -100,18 +125,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "np_array = np.array(np.arange(2.0, 11.0))\n",
```

Comment:
The array `np_array` should have a length between 5 and 10 as per the task requirements. Currently, it has a length of 9. Consider adjusting the range to meet the requirement.